src/main/java/org/ibp/api/rest/location/LocationResource.java

```diff
@@ -106,13 +105,21 @@
 
 	@ApiOperation(value = "Delete Location", notes = "Delete Location")
 	@RequestMapping(value = "/crops/{cropName}/locations/{locationId}", method = RequestMethod.DELETE)
-	@PreAuthorize("hasAnyAuthority('ADMIN', 'CROP_MANAGEMENT', 'MANAGE_PROGRAMS')")
-	@ResponseBody
+	@PreAuthorize("hasAnyAuthority('ADMIN', 'CROP_MANAGEMENT', 'MANAGE_CROP_SETTINGS')")
 	public ResponseEntity<Void> deleteLocation(@PathVariable final String cropName,
 		@PathVariable final Integer locationId,
 		@RequestParam(required = false) final String programUUID) {
 		this.locationService.deleteLocation(locationId);
 		return new ResponseEntity<>(HttpStatus.NO_CONTENT);
 	}
 
+	// Temporaly resource until we remove the country table.
```

Comment:
@cuenyad Temporaly = temporary, please add a reference to https://ibplatform.atlassian.net/browse/IBP-5462